### PR TITLE
switch to using prop-types for React 15.5+ compat

### PR DESCRIPTION
**Pull request by davidascher**
using the codemod.

### CODE REVIEW

File: examples/basic/Count.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import React, { PropTypes } from 'react'
+import PropTypes from 'prop-types';
+import React from 'react';
```
Comment (simenbrekken) [inline examples/basic/Count.js]: @davidascher CircleCI seems to be drunk again but could you please make sure this passes `npm run lint`? Our ESLint config doesn't allow semis.

**Comment by davidascher** [comment]
Ah sure, sorry I missed that

**Comment by einarlove** [comment]
@davidascher, are you still moving forward with this PR?

**Comment by davidascher** [comment]
Sorry, fell off my radar.  hopefully updated PR fixes the lint rules.

**Pull request merged**